### PR TITLE
DCS-718 Home should always be '/' rather than DPS. 

### PR DESCRIPTION
**Pull request by andrewrlee**
Adding graphviz diagram for routes

Diagram looks like:
<kbd>
routes</kbd>

### CODE REVIEW

File: views/autherror.njk [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <div class="govuk-grid-column-two-thirds">
         <h1 class="govuk-heading-l">Authorisation Error</h1>
         <p>There was a problem authorising you to use this application.</p>
-        <p>Please go to <a class="govuk-link govuk-link--no-visited-state" href="{{ authURL }}">here</a> to try again.</p>
+        <p>Please go to <a class="govuk-link govuk-link--no-visited-state" href="/logout">here</a> to try again.</p>
```
Comment (andrewrlee) [inline views/autherror.njk:11]: if problem with auth, get user to log in again

File: helm_deploy/hmpps-book-video-link/templates/_envs.tpl [hunk 2]
```diff
@@ -86,9 +86,6 @@ env:
   - name: OAUTH_ENDPOINT_URL
     value: {{ .Values.env.OAUTH_ENDPOINT_URL | quote }}
 
-  - name: NN_ENDPOINT_URL
```
Comment (andrewrlee) [inline helm_deploy/hmpps-book-video-link/templates/_envs.tpl:89]: no longer required

**Review by Nimphal — APPROVED** [review]
LGTM!

The diagram suggests there's no "no room availability" flow for the request journey - is that a diagram artifact or it is actually missing even though the request journey also checks availability?

**Review by JamieDraperUK — APPROVED** [review]
LGTM

**Pull request merged**